contracts/compliance/ICompliance.sol

```diff
@@ -87,66 +86,73 @@ interface ICompliance {
+     *  @param _amount The amount of tokens involved in the transfer
+     */
+    function transferred(
+        address _from,
+        address _to,
+        uint256 _amount
+    ) external;
 
-   /**
-    *  @dev checks that the transfer is compliant.
-    *  default compliance always returns true
-    *  READ ONLY FUNCTION, this function cannot be used to increment
-    *  counters, emit events, ...
-    *  @param _from The address of the sender
-    *  @param _to The address of the receiver
-    *  @param _amount The amount of tokens involved in the transfer
-    */
-    function canTransfer(address _from, address _to, uint256 _amount) external view returns (bool);
-
-   /**
-    *  @dev function called whenever tokens are transferred
-    *  from one wallet to another
-    *  this function can update state variables in the compliance contract
-    *  these state variables being used by `canTransfer` to decide if a transfer
-    *  is compliant or not depending on the values stored in these state variables and on
-    *  the parameters of the compliance smart contract
-    *  @param _from The address of the sender
-    *  @param _to The address of the receiver
-    *  @param _amount The amount of tokens involved in the transfer
-    */
-    function transferred(address _from, address _to, uint256 _amount) external;
-
-   /**
-    *  @dev function called whenever tokens are created
-    *  on a wallet
-    *  this function can update state variables in the compliance contract
-    *  these state variables being used by `canTransfer` to decide if a transfer
-    *  is compliant or not depending on the values stored in these state variables and on
-    *  the parameters of the compliance smart contract
-    *  @param _to The address of the receiver
-    *  @param _amount The amount of tokens involved in the transfer
-    */
+    /**
+     *  @dev function called whenever tokens are created
+     *  on a wallet
+     *  this function can update state variables in the compliance contract
+     *  these state variables being used by `canTransfer` to decide if a transfer
+     *  is compliant or not depending on the values stored in these state variables and on
+     *  the parameters of the compliance smart contract
+     *  @param _to The address of the receiver
+     *  @param _amount The amount of tokens involved in the transfer
+     */
     function created(address _to, uint256 _amount) external;
 
-   /**
-    *  @dev function called whenever tokens are destroyed
-    *  this function can update state variables in the compliance contract
-    *  these state variables being used by `canTransfer` to decide if a transfer
-    *  is compliant or not depending on the values stored in these state variables and on
-    *  the parameters of the compliance smart contract
-    *  @param _from The address of the receiver
-    *  @param _amount The amount of tokens involved in the transfer
-    */
+    /**
```

Comment:
would have been a good idea to lint in another PR, it's very hard to see the real changes, that are related to the proxy addition....